narwhal/types/src/primary.rs

```diff
@@ -420,12 +420,7 @@ impl Vote {
             committee.stake(&self.author) > 0,
             DagError::UnknownAuthority(self.author.encode_base64())
         );
-
-        // Check the signature.
-        let vote_digest: Digest<{ crypto::DIGEST_LENGTH }> = self.digest().into();
-        self.signature
-            .verify_secure(&to_intent_message(vote_digest), &self.author)
-            .map_err(|_| DagError::InvalidSignature)
+        Ok(())
```

Comment:
perhaps we want to rename this function to validate since it doesnt actually run verify sig? there are few other things it validates though